.github/workflows/draft-release.yml

```diff
@@ -47,6 +47,12 @@ jobs:
         run: |
           ./dbld/rules release VERSION=`cat VERSION`
 
+      - name: Store release tarball as artifact
```

Comment:
The error handling can be improved a bit.
The problems are:
 * The comment to the Version Bump PR is sent prematurely.
 * The draft release is created before trying to upload the packages to the APT repo. The upload could fail and we are left with a draft release which will have to be removed manually.

I propose something like this:
 * `create-release-tarball` job, which uploads the tarball as an artifact
 * `packages` job, which needs the `create-release-tarball` job, and generates and uploads deb packages to the APT repo
 * `draft-release` job, which needs the `create-release-tarball` and `packages` jobs, and it creates the draft release t o Github
 * `comment-workflow-status` job, which needs the `packages` and `draft-release` jobs, and it comments on the Version Bump PR, merging the 2 results.